pkgs/development/python-modules/airlift/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
+
+  disabled = pythonOlder "3.9";
+  pname = "airlift";
+  format = "pyproject";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "jl178";
+    repo = "airlift";
+    rev = "v${version}";
+    sha256 = "ZQ+YCMXQBM5Gb5O1jDQfKBeS1WxFKrtSRelgj65+f9U=";
+  };
+  nativeBuildInputs = [ poetry-core ];
+  propagatedBuildInputs = [
+    argparse
+    halo
+    pyyaml
+    hiyapyco
+    jinja2
+    dotmap
+    requests
+    kubernetes-helm
+    kind
+    docker
+  ];
+  meta = with lib; {
+    description =
+      "A flexible, configuration driven CLI for Apache Airflow local development";
+    homepage = "https://github.com/jl178/airlift";
```

```suggestion
    homepage = "https://github.com/jl178/airlift";
    changelog = "https://github.com/jl178/airlift/releases/tag/${src.rev}";
```
Adding changelog is useful for future updates.